CONTRIBUTING.md

```diff
@@ -114,14 +114,9 @@ This section provides a step-by-step guide to set up and start working on the pr
 
 To begin, install prerequisites:
 
-`wget` is required by the maven installer
+`python` is the runtime language for remorph
 ```shell
-brew install wget
-```
-
-`python` is the dependency manager for JVM based languages
-```shell
-brew install maven
+brew install python
```

Comment:
Have we stopped telling contributors to run `make dev` because hatch sets up a .venv for python and virtual environments are better than `brew install python`. Especially as the project seems to require 3.10.x (or maybe it does not now?).

Reply:
we need to fix our contributing guide after all the changes over the holidays on installing Python. and here `brew install` will limit to macos need to address wider community.